.github/workflows/nodejs.yml

```diff
@@ -11,40 +11,45 @@ on:
       - master
 
 jobs:
-  build:
+  test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         node-version: [12.x]
     steps:
     - uses: actions/checkout@master
+
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: npm install
+
+    - name: Install npm
       run: |
         npm version
-        npm install -g npm
-        npm version
-        npm install
+        npm ci
       env:
         CI: true
-    - name: End-to-end Tests with Cypress
+
+    - name: Tests
       uses: cypress-io/github-action@v1
```

Comment:
Since we are now using the `npm test` script, do we still need to use the `cypress-io` action? Seems redundant.

Also @BKristenssonAlfsson brought up a valid potential issue with that. Using the github action introduces a difference between running cypress tests on a local dev environment and CI. Hence the use of `npm test`.

Reply:
I think we need @BKristenssonAlfsson 's help

Reply:
@ivelin @sniperadmin I tried to remove the cypress-io action. It generated more errors in the CI pipe, so I kept it for now. Might be removable now when Nasr fixed it all.

Reply:
OK, let's try to fix that so we have consistent behavior in all environments.

Reply:
One of the problems is that cypress-io has wait-on which is needed to make sure the UI is started